libs/payments/content-server/src/lib/content-server.config.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { IsString } from 'class-validator';
+import { faker } from '@faker-js/faker';
+import { Provider } from '@nestjs/common';
+
+export class ContentServerClientConfig {
+  @IsString()
```

Comment:
nit. Could this be updated to use `@IsUrl`?

Reply:
yes, if we allow { require_tld: false } for local development